packages/graphql/commands/graphql.js

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env node
+'use strict';
+
+var fetchFragments = require('../lib/fragments');
+
+module.exports = function defineGraphQLCommand (args) {
+  args.command('graphql', 'Execute GraphQL specific tasks', function (yargs) {
+    return yargs
+      .usage('Usage: hops-graphql <command>')
```

Comment:
Why not use `$0` - so that the usage description changes depending on whether you call the binary directly or through the local cli?